config/browsers.json

```diff
@@ -1,6 +1,6 @@
 {
   "firefox": "28",
-  "chrome": "23",
-  "chromium": "23",
+  "chrome": "38",
+  "chromium": "38",
```

Comment:
Chrome 38 is the lowest version that's been used with any frequency since January, according to analytics. It's also a version that bumped up a lot of ES6 standard library support.